indexer/indexer.py

```diff
@@ -16,6 +16,8 @@
 webifydir = "/webify"
 vulnixfiles = "vulnix*.txt"
 resultfiles = "*_results/**/*.html"
+sbomfiles = "sbom.runtime*"
+vulnxsfiles = "vulns.runtime*.csv"
```

Comment:
If we ever want to add buildtime sboms/vulnxscan reports, these patterns will not find matches to those.
You could just accept all "sbom.*" and "vulns.*.csv", or this can also be changed later if we add the buildtime stuff too.

Reply:
You're correct. I think this is fine for now, I'll update that when buildtime reports are added.